Makefile

```diff
@@ -177,3 +177,11 @@ ifndef GITHUB_AUTH_TOKEN
 	$(error GITHUB_AUTH_TOKEN is undefined)
 endif
 ###############################################################################
+
+
+###############################################################################
+
+.PHONY: ko-local
+ko-local:
+	KO_DOCKER_REPO=${KO_PREFIX}/scorecard CGO_ENABLED=0 ko publish --bare --local --tags stable --tags $(GIT_HASH) .
```

Comment:
You can just set KO_DOCKER_REPO, you only need the prefix bit if you have multiple images.

Reply:
The reason I am prefixing it was we do have multiple images. These images are in the cron-job. Thanks